MainApp/AssemblyOne/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace AssemblyOne
+{
+    public class AssemblyOneType
+    {
+        private int privateInt = 1;
+        public string publicString = "Hello";
+        protected bool protectedBool = true;
+        internal const string InternalConst = "Nick";
+        protected internal int protectedInternalVal = 2020;
+        protected private char protectedPrivate()
```

Comment:
methods should be started from Upper case:
protected private char **P**rotectedPrivate()